tests/nimble_autoconn_gnrc/Makefile

```diff
@@ -0,0 +1,23 @@
+BOARD ?= nrf52dk
+include ../Makefile.tests_common
+
+# include shell support
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Include GNRC and RPL
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_ipv6_router_default
+USEMODULE += gnrc_icmpv6_echo
+USEMODULE += auto_init_gnrc_rpl
+USEMODULE += gnrc_rpl
+
+# Setup Nimble
+USEMODULE += nimble_autoconn_ipsp
+NIMBLE_MAX_CONN := 3
```

Comment:
Does this need to be `:=`

Reply:
probably not, but also it does not hurt I guess. But I can just use `=` if you like it better...